financials/reports/funding_transparency_2019Q1.md

```diff
@@ -0,0 +1,90 @@
+# 2019Q1 Funding Transparency Report
+
+Last updated: May 02 2019
+
+## Introduction
+This document provides a high level overview of Grin's financials for Q1 of 2019 (Jan 01 - Mar 31). The purpose is to keep the community up to date on funding activity in transparent and digestible way. This is the first attempt at such a report, and is based on financial logs that was introduced during the quarter.
+
+### Notes
+
+1. Data was up to date at the time of writing, but it's still possible that there have been changes and updates in the logs since.
+2. USD conversion rates are derived from the closing price of the asset on the day in question according to Coinmarketcap.com unless otherwise stated. 
+
+
+
+## Status before the period
+
+At the end of Dec 31 2018, Grin held the equivalent of $65,237.35 in the following assets:
+
+Currency | Amount | USD x-rate Dec 31 2018 | USD Equivalent | Wallet address(es)
+|---|---:|---:|---:|---|
+BTC | 17.34294023 | 3,742.70 | $64,909.42 | 3ChVP627KU5w4zu2rieFPF3wGXWQgmhvrs <br />bc1qdgs8vkpzr256qjlzlfht72z3mhcrdrt6wj2rfjw39j8us24gz8uq78qj65
```

Comment:
For ease of review, the BTC sum here can be verified in the following links:

https://blockchair.com/bitcoin/outputs?q=is_spent(false),recipient(bc1qdgs8vkpzr256qjlzlfht72z3mhcrdrt6wj2rfjw39j8us24gz8uq78qj65),time(..2018-12-31)&s=recipient(desc),value(desc)#
https://api.blockcypher.com/v1/btc/main/addrs/3ChVP627KU5w4zu2rieFPF3wGXWQgmhvrs?before=556458

There is room for automating the generation of these numbers from the income log and verification using public apis.

Reply:
That's right! Once we are happy with the format, it would be nice to have these reports automated moving forward. Would also be cool if we could auto-monitor our addresses and automatically insert income_log / spending_log entries the moment there is activity. Would you like to take a stab at doing any of that? Or perhaps open issues for them under /grin-pm/ ?

Reply:
Opened https://github.com/mimblewimble/grin-pm/issues/128 for the automated generation of these reports, not sure about your auto-monitor proposal, please go ahead and articulate that in its own issue if you wish.